app/front/scripts/controllers/main.js

```diff
@@ -69,7 +73,24 @@ angular.module('Application')
       function initRegular() {
         $q(LoginService.tryGetToken())
           .then(function(token) {
-            return $q(osViewerService.loadDataPackages(token));
+            var packageId = null;
+            var userId = token ? LoginService.getUserId() : null;
+            if (!token) {
```

Comment:
Okay, so you see the problem there - loggedIn users should see their packages in the dropdown, but still be able to see any other public package (based on the URL).